MeesGame/UI/Controls/EditorControls/ColorPicker.cs

```diff
@@ -10,26 +13,18 @@ namespace MeesGame
     class ColorPicker : UIComponent
     {
         private const int width = 200;
-        private const int height = 200;
-        private const int colorsPerRow = 4;
+        private const int height = 250;
```

Comment:
The height changes the height of the entire colorpicker, but the height is not related to the text because that height is determined by the size of the text (this can be changed by changing the font).